### PR TITLE
YCDOCIO-2018: Data transfer MySQL MCH

### PR DESCRIPTION
**Pull request by voloschuk-fedor**
https://st.yandex-team.ru/YCDOCIO-2018

### CODE REVIEW

File: tutorials/terraform/data-transfer/data-transfer-mmy-mch.tf [hunk 1]
```diff
@@ -0,0 +1,235 @@
+# Infrastructure for the Yandex Cloud Managed Service for MySQL, Managed Service for ClickHouse and Data Transfer.
```
Comment (redalka-cloud) [inline tutorials/terraform/data-transfer/data-transfer-mmy-mch.tf:1]: Infrastructure for the Yandex Cloud Managed Service for MySQL, Managed Service for ClickHouse, and Data Transfer.

File: tutorials/terraform/data-transfer/data-transfer-mmy-mch.tf [hunk 2]
```diff
@@ -0,0 +1,235 @@
...
+
+  ingress {
+    description    = "Allow connections to the Managed Service for MySQL cluster from the Internet"
+    protocol       = "TCP"
+    port           = 3306
+    v4_cidr_blocks = ["0.0.0.0/0"]
+  }
+
+  ingress {
+    description    = "Allow connections with clickhouse-client to the Managed Service for ClickHouse cluster from the Internet"
```
Comment (redalka-cloud) [inline tutorials/terraform/data-transfer/data-transfer-mmy-mch.tf:63]: Allow connections with a ClickHouse client to the Managed Service for ClickHouse cluster from the Internet

File: tutorials/terraform/data-transfer/data-transfer-mmy-mch.tf (hunk 1 above)
Comment (AnatolyKraev) [reply tutorials/terraform/data-transfer/data-transfer-mmy-mch.tf:1]: Ок

File: tutorials/terraform/data-transfer/data-transfer-mmy-mch.tf (hunk 2 above)
Comment (AnatolyKraev) [reply tutorials/terraform/data-transfer/data-transfer-mmy-mch.tf:63]: Ок

**Review by redalka-cloud — APPROVED** [review]

**Pull request merged**